google/cloud/security/inventory/inventory_loader.py

```diff
@@ -18,7 +18,7 @@
 
 Usage:
   $ forseti_inventory \\
-      --inventory_groups <true|false> (optional) \\
+      --pipeline_config (required) \\
       --groups_service_account_key_file <path to file> (optional)\\
```

Comment:
That's fine. We'll need to adjust the GCP deployment script to account for this though.